spring-auto-restdocs-core/src/main/java/capital/scalable/restdocs/payload/AbstractJacksonFieldSnippet.java

```diff
@@ -87,13 +90,37 @@ protected FieldDescriptors createFieldDescriptors(Operation operation,
 
     protected Type firstGenericType(MethodParameter param) {
         Type type = param.getGenericParameterType();
-        if (type instanceof ParameterizedType) {
+        if(type instanceof TypeVariable) {
```

Comment:
Your formatting is a bit inconsistent with regards to space after `if` or `for`. In this project, we always place a space behind it.

Reply:
Updated formatting